pandas-stubs/core/indexes/frozen.pyi

```diff
@@ -0,0 +1,18 @@
+from typing import Generic, Iterator, List, overload, TypeVar, Tuple, Union
+
+__all__ = ["FrozenList"]
```

Comment:
you don't really need this, because the things you import from typing are not re-exported (they're not using the `as ...` import) and the variable `_T` starts with an underscore, so that's also not exported.

Reply:
I would slightly prefer if you removed all these `__all__`, they're not used anywhere else in these type stubs